features/json-raw.yml

```diff
@@ -0,0 +1,8 @@
+name: JSON source text access
+description: The `rawJSON` value handling in `JSON.stringify()` and the source context parameter for the `JSON.parse()` reviver callback serialize and parse JSON in a lossless way.
```

Comment:
Optional suggestion:

Now that we've settled on a direction for this, may I suggest a change to avoid the garden path sentence?  Stacking a bunch of verbs in a row—"reviver callback seralize and parse"—makes it quite hard to make sense of. I'm suggesting departing from our usual front-loading style here, to be more readable:

```suggestion
description: To serialize and parse JSON in a lossless way, `JSON.stringify()` handles `rawJSON` values and `JSON.parse()`'s reviver callback takes a source context parameter.
```